lib/rubocop/cop/rails/bulk_change_table.rb

```diff
@@ -207,6 +207,18 @@ def database_yaml
           nil
         end
 
+        def database_from_env
+          url = ENV['DATABASE_URL'].presence
+          return nil unless url
+
+          case url
+          when %r{\Amysql2://}
+            MYSQL
+          when %r{\Apostgres(ql)?://}
```

Comment:
Rails normalizes `postgres` to `postgresql` in https://github.com/rails/rails/blob/f517eefe1450b900cf3e44b8e6830d917f1530c0/activerecord/lib/active_record/database_configurations/connection_url_resolver.rb#L29